docs/elements/hx-tabset/index.html

```diff
@@ -1,18 +1,42 @@
 ---
 title: <hx-tabset>
+also:
+  components/tabs: Tabs
+  elements/hx-tab: <hx-tab>
+  elements/hx-tabcontent: <hx-tabcontent>
+  elements/hx-tablist: <hx-tablist>
+  elements/hx-tabpanel: <hx-tabpanel>
 ---
 {% extends 'element.njk' %}
 {% block content %}
+  <section>
+    <p>
+      The custom <code>&lt;hx-tabset&gt;</code> element provides functionality to
+      navigate between one or more tab panels.
+    </p>
 
-<section>
-  <h2 class="hxSectionTitle" id="attributes">Attributes</h2>
+    <hx-dl class="hxBox-md metadata">
+      <hx-def>
+        <hx-dt>Permitted Parents</hx-dt>
+        <hx-dd>any</hx-dd>
+      </hx-def>
+      <hx-def>
+        <hx-dt>Permitted Children</hx-dt>
+        <hx-dd>
+          one of each:
+          <ul>
+            <li>an <a href="elements/hx-tablist">&lt;hx-tablist&gt;</a> element</li>
+            <li>an <a href="elements/hx-tabcontent">&lt;hx-tabcontent&gt;</a> element</li>
+          </ul>
+        </hx-dd>
+      </hx-def>
+    </hx-dl>
+  </section>
+{% endblock %}
+
+{% block attributes %}
   <dl>
-    <dt>
-      current-tab <small>{Number=0}</small>
-    </dt>
-    <dd>
-      Zero-based index of the currently active tab + tabpanel.
-    </dd>
+    <dt>current-tab <small>{Number=0}</small></dt>
+    <dd>Zero-based index of the currently active tab + tabpanel.</dd>
```

Comment:
yeah some `<dd>` are on one line and others on seperate (even though they can fit on one)- we should fix in another story to cleanup